roles/ansible/tower/config-ansible-tower/README.md

```diff
@@ -18,6 +18,7 @@ The variables used to install an Ansible Tower instance are outlined in the tabl
 |:---------|:------------|:---------|:---------|
 |ansible_tower_download_url|URL from which the Tower installer will be downloaded. If different version required, recommended to change `ansible_tower_version`|no|'https://releases.ansible.com/ansible-tower/setup/ansible-tower-setup-{{ ansible_tower_version }}.tar.gz'|
 |ansible_tower_version|Version of Ansible Tower to install|no|'3.3.0-1'|
+|ansible_tower_oc_download_url|Version of oc client to install. If no URL specified the default will be used.|no|'https://mirror.openshift.com/pub/openshift-v3/clients/3.10.47/linux/oc.tar.gz'|
```

Comment:
Tthe "If no URL specified ..." part is implied with this table having a default column. To keep it simple I'd recommend removing that sentence.